test/cli/correct-bare-stdlib-generics/correct-bare-stdlib-generics.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+tmp="$(mktemp -d)"
+infile="test/cli/correct-bare-stdlib-generics/correct-bare-stdlib-generics.rb"
+cp "$infile" "$tmp"
+
+cwd="$(pwd)"
+cd "$tmp" || exit 1
+
+"$cwd/main/sorbet" --silence-dev-message -a correct-bare-stdlib-generics.rb 2>&1
+
+echo
+echo --------------------------------------------------------------------------
+echo
+
+cat correct-bare-stdlib-generics.rb
+
+rm correct-bare-stdlib-generics.rb
+rm "$tmp"
```

Comment:
`$tmp` is a directory, so I think you need to do this instead
```suggestion
rm -rf "$tmp"
```